### PR TITLE
Fix failing workflows

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
 - Fixes several typos in the GitHub actions
 - Rolls out support for the legacy 'views-api'

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- Wrong configmap in staging workflow
image

- Wrong build context in network-uptime deploy preview
image

- Obsolete deploy to pipeline cluster
image

- View api breaking deployment patch
image

- Use of wrong manifest file for production deployment
image

**_HOW DO I TEST OUT THIS PR?_**
- Static code analysis

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

### CODE REVIEW

File: .github/workflows/deploy-apis-to-staging.yml [hunk 1]
```diff
@@ -1376,20 +1376,7 @@ jobs:
...
-          kubectl set image deployment/airqo-stage-calibrate-api sta-calibrate=${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-calibrate-api:${{ needs.image-tag.outputs.build_id }} -n staging
-          kubectl annotate deployment/airqo-stage-calibrate-api kubernetes.io/change-cause="Image updated to ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-calibrate-api:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n staging
-
-      - name: Login to Pipeline k8s cluster
-        uses: azure/k8s-set-context@v1
-        with:
-          method: kubeconfig
-          kubeconfig: ${{ secrets.K8S_PIPELINE_CONFIG }}
-
-      - name: Deploy to Pipeline k8s cluster
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-staging.yml:1390]: Remove deploy to pipeline cluster

File: .github/workflows/deploy-apis-to-production.yml [hunk 2]
```diff
@@ -1134,10 +1134,9 @@ jobs:
       - name: Update the corresponding k8s configmap(s)
         run: |
           cd src/view/
-          gcloud secrets versions access latest --secret="sta-env-view-api" > api.env
+          gcloud secrets versions access latest --secret="prod-env-view-api" > api.env
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:1137]: [correct secret names for view and analytics](https://github.com/airqo-platform/AirQo-api/pull/1062/commits/e1fa6cb672baebaf9a00c6aa2ffe5f6e721ba57c)

File: k8s/view-api/prod-view-api.yaml [hunk 3]
```diff
@@ -70,38 +70,8 @@ spec:
           lifecycle:
             preStop:
               exec:
-                command: [ "sh", "-c", "sleep 10" ]
-        - name: views-api
```
Comment (123MwanjeMike) [inline k8s/view-api/prod-view-api.yaml:74]: [rollout support for 'views-api'](https://github.com/airqo-platform/AirQo-api/pull/1062/commits/df8cd5c1db242f7308433d940c0c8d77f3dd3f9f)

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**